examples/microGPT.py

```diff
@@ -28,7 +28,7 @@ def __init__(
         vocab_size,
         weight_decay=0.1,
         betas=(0.9, 0.95),
-        learning_rate=6e-4,
+        learning_rate=1e-4,
```

Comment:
before this PR favor cannot run in fp16 (the very first FW dies). With this PR it runs just fine, but it requires a LR a little lower than other mechanisms to prevent overflow later on